app/templates/index.hbs

```diff
@@ -32,19 +31,30 @@
   {{/splash-weather}}
 </div>
 <div class="ui container">
-  <h1>Why so hot?</h1>
-  <p><strong>Pellentesque habitant morbi tristique</strong> senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. <em>Aenean ultricies mi vitae est.</em> Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, <code>commodo vitae</code>, ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. <a href="#">Donec non enim</a> in turpis pulvinar facilisis. Ut felis.</p>
-</div>
-<div class="black">
-  <div class="ui container">
-    <h1>Stay safe!</h1>
-    <p><strong>Pellentesque habitant morbi tristique</strong> senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. <em>Aenean ultricies mi vitae est.</em> Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, <code>commodo vitae</code>, ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis tempus lacus enim ac dui. <a href="#">Donec non enim</a> in turpis pulvinar facilisis. Ut felis.</p>
+  <h2>Why is it so hot?</h2>
+  <p>
+    Why does it seem to be getting hotter? Learn the science behind the hot weather.
+  </p>
+  <div style="text-align: center;">
+    <a href="/info/why-is-it-so-hot" class="ui red basic button">
```

Comment:
I missed this, but for future reference, all hyperlinks should use the hbs {{#link-to}} helper so that the router is properly configured. Otherwise the native href triggers a hard reload. Thanks!